### DISCUSSION TITLE
Configure docker compose

### DISCUSSION THREAD

**Discussion by HanaO00** [louislam/dockge#590] [Ask for Help]
### ⚠️ Please verify that this bug has NOT been raised before.

- [X] I checked and didn't find similar issue

### 🛡️ Security Policy

- [X] I agree to have read this project [Security Policy](https://github.com/louislam/dockge/security/policy)

### 📝 Describe your problem

Hello I have a little trouble with the docker compose, using https://dockge.kuma.pet/ f i put my path, the install script below always uses this 
`# Create directories that store your stacks and store Dockge's stack
1. mkdir -p /opt/stacks /opt/dockge
2. cd /opt/dockge`

But can I place the dockge stack wherever I want using also the ./data or do I have to use /opt/ ?

My compose looks like this

services:
  dockge:
    image: louislam/dockge:1
    restart: unless-stopped
    ports:
      - 5001:5001
    volumes:
      - /var/run/docker.sock:/var/run/docker.sock
      - ./data:/app/data
      # Stacks Directory
      # ⚠️ READ IT CAREFULLY. If you did it wrong, your data could end up writing into a WRONG PATH.
      # ⚠️ 1. FULL path only. No relative path (MUST)
      # ⚠️ 2. Left Stacks Path === Right Stacks Path (MUST)
      - /home/myuser/docker:/home/myuser/docker
    environment:
      # Tell Dockge where to find the stacks
      - DOCKGE_STACKS_DIR=/home/myuser/docker

And the dockge stack is in /home/myuser/docker/dockge/compose.yaml

### 🐻 Dockge Version

1

### 💻 Operating System and Arch

Debian

### 🌐 Browser

Firefox

**Comment by cyril59310** [comment]
yes juste remplace container volume  
`- ./data : /app/data` for `/home/myuser/docker/dockge:/app/data`

**Comment by HanaO00** [comment]
Thanks for your help
Maybe `/home/myuser/docker/dockge/data:/app/data` is better or it will cause trouble ?

**Reply by cyril59310** [reply]
Yes, you can separate your Compose file from the configuration files, this will not cause any issues
